### PR TITLE
HAI-2116 Added central business area tormays material

### PR DESCRIPTION
**Pull request by petri-timlin**
# Description

* Compute central business area tormays materials base on named attributes

* Implemeted test mixin method for multiple geometries

* Document updated for central business areas processing

* Update scripts/gis-material-update/data/README.md

### Jira Issue: 
https://helsinkisolutionoffice.atlassian.net/browse/HAI-2116

## Type of change

- [ ] Bug fix 
- [x] New feature 
- [ ] Other

# Instructions for testing
Check instructions from scripts/gis-material-update/readme to process central_business_area data. 
Then check data (=tormays_central_business_areas.gpkg) with QGIS:
* current feature count: 123 (this amount might be changed over time)
* geometries are looking ok (=areas)
* objects are having following attributes: 
                "fid",
                "central_business_area",
                "tunnus",
                "osaalue_tunnus",
                "osaalue_nimi_fi",
                "osaalue_nimi_se",
                "peruspiiri_nimi_fi",
                "peruspiiri_nimi_se",
                "suurpiiri_nimi_fi",
                "suurpiiri_nimi_se"

# Checklist:

- [x] I have written new tests (if applicable)
- [x] I have ran the tests myself (if applicable)
- [x] I have made necessary changes to the documentation, link to confluence
 or other location: 
https://helsinkisolutionoffice.atlassian.net/wiki/spaces/HAI/pages/8292401164/Paikkatietoaineiston+k+sittely#Kantakaupunki

# Other relevant info
Please describe here if there is e.g. some requirements for this change or
 other info that the tester/user needs to know.

### CODE REVIEW

**Review by umeetiusbaar — APPROVED** [review]
Tested by running:
```
$ cd scripts/gis-material-update/
$ docker volume create --name=haitaton_gis_prepare
$ docker volume create --name=haitaton_gis_db
$ docker-compose build
$ sh copy-files.sh
$ docker-compose run --rm gis-fetch central_business_area 
$ docker-compose up -d gis-db
$ docker-compose run --rm gis-fetch central_business_area
$ docker-compose run --rm gis-process central_business_area
$ docker-compose stop gis-db
$ docker run --rm --network=haitaton-backend_backbone -v $(pwd)/haitaton-gis-output:/tmp osgeo/gdal:alpine-small-latest ogr2ogr -overwrite -f PostgreSQL PG:"dbname='haitaton' host='db' port='5432' user='****' password='****'" /tmp/tormays_central_business_areas.gpkg
```
The last step needs to have local db up and running.
After this I checked with QGIS that there is new db table `tormays_central_business_areas` (old one had `_polys` suffix) and it contains the same geometries as the old one had (or so it seems).

File: scripts/gis-material-update/config.yaml [hunk 1]
```diff
@@ -62,6 +62,11 @@ cycle_infra:
   target_buffer_file: "tormays_cycle_infra_polys.gpkg"
   buffer:
     - 20
+central_business_area:
+  addr: "WFS:https://kartta.hel.fi/ws/geoserver/avoindata/wfs"
+  layer: "avoindata:Piirijako_pienalue"
```
Comment (corvidian) [inline scripts/gis-material-update/config.yaml:67]: Is there a layer for osaalue? That's the lowest level we care about.

**Comment by corvidian** [comment]
> there is new db table `tormays_central_business_areas` (old one had `_polys` suffix)

If the table name changes, we have to make changes to the Kotlin side to use the new data. The transition would be smoother if the name of the table stayed the same.

File: scripts/gis-material-update/config.yaml (hunk 1 above)
Comment (corvidian) [reply scripts/gis-material-update/config.yaml:67]: I'll answer myself: Yes there is, but it doesn't have peruspiiri data, so we can't filter it.

Comment (petri-timlin) [reply scripts/gis-material-update/config.yaml:67]: Osaalue(=osaalue_nimi_fi) and peruspiiri(=peruspiiri_nimi_fi) are tormays_central_business_areas attribute fields

**Comment by petri-timlin** [comment]
> > there is new db table `tormays_central_business_areas` (old one had `_polys` suffix)
> 
> If the table name changes, we have to make changes to the Kotlin side to use the new data. The transition would be smoother if the name of the table stayed the same.

This was my mistake. Table name should not change at all.

**Review by corvidian — APPROVED** [review]

**Pull request merged**